### PR TITLE
update gsuite title

### PR DESCRIPTION
**Pull request by lindsey-w**
### Background

Add document info to the alert title to make it more useful.

### Changes

* add `doc_title` to alert title
* linter did some things

### Testing

* `make ci`

### CODE REVIEW

**Review by wey-chiang — APPROVED** [review]

**Comment by nhakmiller** [comment]
@lindsey-w would it be possible to go through the other G Suite rules that pertain to documents and update the titles to include the document name as appropriate? I'm thinking:

- gsuite_doc_ownership_transfer
- gsuite_drive_external_share
- gsuite_drive_overly_visible
- gsuite_drive_visibility_change
-

**Comment by lindsey-w** [comment]
> * gsuite_doc_ownership_transfer

I don't think this one has the document title in the event (based on the [documentation](https://developers.google.com/admin-sdk/reports/v1/appendix/activity/admin-docs-settings#TRANSFER_DOCUMENT_OWNERSHIP))

> * gsuite_drive_external_share

Looks like this one has the title in it ✅ 

```
GSuite.Drive.ExternalFileShare
        [PASS] Dangerous Share of Known Document with a Missing User
                [PASS] [title] Dangerous file share by [example@acme.com]: "Document Title Primary" to outside@acme.com
        [PASS] Dangerous Share of Unknown Document
                [PASS] [title] Dangerous file share by [example@acme.com]: "Untitled document" to alice@external.com
        [PASS] Share Allowed by Exception
```

> * gsuite_drive_overly_visible

✅ 

```
GSuite.DriveOverlyVisible
        [PASS] Access Event
        [PASS] Modify Event Without Over Visibility
        [PASS] Overly Visible Doc Modified
                [PASS] [title] User [bobert@example.com] modified a document [my shared document] that has overly permissive share settings
```

> * gsuite_drive_visibility_change

✅ 

```
GSuite.DriveVisiblityChanged
        [PASS] Access Event
        [PASS] ACL Change without Visiblity Change
        [PASS] Doc Became Public
                [PASS] [dedup] bobert@example.com
                [PASS] [title] User [bobert@example.com] made a document [my shared document] externally visible for the first time
        [PASS] Doc Became Private
```

File: gsuite_reports_rules/gsuite_drive_overly_visible.py [hunk 1]
```diff
@@ -24,10 +24,13 @@ def rule(event):
                                           'visibility') in PERMISSIVE_VISIBILITY
 
 
-def dedup(event):
-    return deep_get(event, 'actor', 'email')
-
-
```
Comment (lindsey-w) [inline gsuite_reports_rules/gsuite_drive_overly_visible.py:30]: removed dedup so it uses the title, which includes username + document title

Comment (lindsey-w) [reply gsuite_reports_rules/gsuite_drive_overly_visible.py:30]: And... I just saw why it this was using email vs email + title.  I can revert.

**Review by nhakmiller — APPROVED** [review]

**Pull request merged**